packages/builders/scripts/inter-protocol/replace-electorate-core.js

```diff
@@ -95,19 +76,38 @@ const configurations = {
 
 const { keys } = Object;
 const Usage = `agoric run replace-electorate-core.js ${keys(configurations).join(' | ')}`;
-export default async (homeP, endowments) => {
-  const { scriptArgs } = endowments;
-  const variant = scriptArgs?.[0];
-  const config = configurations[variant];
+/** @type {import('@agoric/deploy-script-support/src/externalTypes.js').CoreEvalBuilder} */
+export const defaultProposalBuilder = async ({ publishRef, install }, opts) => {
+  const config = configurations[opts.variant];
   if (!config) {
     console.error(Usage);
     process.exit(1);
   }
-  console.log('replace-committee', scriptArgs, config);
+  return harden({
+    sourceSpec: '@agoric/inter-protocol/src/proposals/replaceElectorate.js',
+    getManifestCall: [
+      getManifestForReplaceAllElectorates.name,
+      {
+        ...config,
+        economicCommitteeRef: publishRef(
+          install(
+            '@agoric/governance/src/committee.js',
+            '../bundles/bundle-committee.js',
+          ),
+        ),
+      },
+    ],
+  });
+};
+
+export default async (homeP, endowments) => {
+  const { scriptArgs } = endowments;
+  const variant = scriptArgs?.[0];
```

Comment:
Does this still work using `agoric run` from the command line?

I wonder whether/how it works in bootstrap tests.

Reply:
it runs successfully with agoric run
```
agoric: run: running /Users/frazarshad/Projects/agoric-sdk/packages/builders/scripts/inter-protocol/replace-electorate-core.js
agoric: run: Deploy script will run with Node.js ESM
replace-committee [ 'A3P_INTEGRATION' ] A3P_INTEGRATION
bundle-source --cache-js /Users/frazarshad/Projects/agoric-sdk/packages/builders/scripts/bundles --elide-comments /Users/frazarshad/Projects/agoric-sdk/packages/governance/src/committee.js committee
creating replace-committee-A3P_INTEGRATION-permit.json
creating replace-committee-A3P_INTEGRATION.js
You can now run a governance submission command like:
  agd tx gov submit-proposal swingset-core-eval replace-committee-A3P_INTEGRATION-permit.json replace-committee-A3P_INTEGRATION.js \
    --title="Enable <something>" --description="Evaluate replace-committee-A3P_INTEGRATION.js" --deposit=1000000ubld \
    --gas=auto --gas-adjustment=1.2
Remember to install bundles before submitting the proposal:
  agd tx swingset install-bundle @/Users/frazarshad/.agoric/cache/b1-0e96f699324a7eb95f402d47dff2d7f549b2834b3bddddd0b5ce60256b1e755aa2836159f3be842ba647fb912fa25753ef6919e2d6146ca82ede5ef979f37202.json
  agd tx swingset install-bundle @/Users/frazarshad/.agoric/cache/b1-81fefc654342c6e1ecf570dbc69dd0ae052b6e8aacf9a093feeb64b6348e21e470db8d204924a4e1625b88792da2c2eb84eb5e4d5a1fccc9308ea9381752d9ba.json
```

The bootstrap tests also work as expected given that all bootstrap tests pass successfully